CONTRIBUTING.md

```diff
@@ -96,3 +98,9 @@ Lowercase is preferred for file and folder names, one exception to this is Files
 - `src/components` will contain reusable UI component for eg. `Button`, `Input` etc.
 - If a style is specific only to the component, and cannot be used anywhere else, add it to `<ComponentName>.styles.ts` in the same folder as the component.
 - All css properties that require number as input should be in multiple of 4 and should be used from `src/common/constants/cssScale.ts`.
+
+### Testing
+
+- Write test cases for each component in their directory for e.g. `src/Home/Home.test.tsx` you can also refer to directory structure.
+- Make sure you cover at least 70% test-case scenario.
+- You can check coverage of your test cases with `yarn test:coverage` command.
```

Comment:
We would want to only check coverage for new changes (changes in the current PR), please update the documentation with instructions for the same.

Reply:
*bump*

Clarifying: We want to document the instructions to generate "test coverage" for only the modified files.